### PR TITLE
Added more 3.10 release notes content

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

File: release_notes/ocp_3_10_release_notes.adoc [hunk 1]
```diff
@@ -175,6 +179,43 @@ spec:
   storageClassName: snapshot-promoter
 ----
 
+[[ocp-310-new-storage-provisioners]]
+==== New Storage Provisioners (Technology Preview)
+
+New Storage Provisioners (CephFS) are currently in
```
Comment (vikram-redhat) [inline release_notes/ocp_3_10_release_notes.adoc]: @tkatarki - should we be removing this from the release notes as per your email?

@ahardin-rh @aheslin

**Comment by vikram-redhat** [comment]
@ahardin-rh - if you rebase this branch against master, your build should pass.

**Comment by vikram-redhat** [comment]
@ahardin-rh - can you please take out the CNV content from the release notes? 

@ousleyp - if I understand that CNV team doesn't want this content highlighted, please correct me if I am wrong.

@aheslin

**Comment by ahardin-rh** [comment]
Applied some edits in response to final review feedback. Merging and can follow-up with further edits if necessary.

File: release_notes/ocp_3_10_release_notes.adoc [hunk 2]
```diff
@@ -414,8 +495,6 @@ inspect them using normal `oc` CLI commands.
 move to RPMs or the kubelet system containers (by setting
 `openshift_use_system_containers=true`), to start or run the platform.
```
Comment (ganhuang) [inline release_notes/ocp_3_10_release_notes.adoc]: `openshift_use_system_containers` is removed, the transition will be done automatically per the kind of operation during the upgrade.

File: release_notes/ocp_3_10_release_notes.adoc [hunk 3]
```diff
@@ -414,8 +495,6 @@ inspect them using normal `oc` CLI commands.
...
 
-* `openshift_node_bootstrap` is now removed. All nodes must be bootstrapped.
-
 * openshift-sdn and openvswitch, if enabled, will be run in a daemonset.
```
Comment (ganhuang) [inline release_notes/ocp_3_10_release_notes.adoc]: I can't understand what does it mean "if enabled" (I might be wrong to get it). In 3.10, openshift-sdn and openvswitch will definitely be run as daemonset, no option to disable or enable.

File: release_notes/ocp_3_10_release_notes.adoc (hunk 2 above)
Comment (ahardin-rh) [reply release_notes/ocp_3_10_release_notes.adoc]: @ganhuang Thanks for your review. This is addressed in https://github.com/openshift/openshift-docs/pull/11106

File: release_notes/ocp_3_10_release_notes.adoc (hunk 3 above)
Comment (ahardin-rh) [reply release_notes/ocp_3_10_release_notes.adoc]: @ganhuang Thanks! This is also addressed in https://github.com/openshift/openshift-docs/pull/11106

**Comment by chengzhang1016** [comment]
@ahardin-rh 
Below comments from Service Catalog & ASB QE teams, thanks.
1. Should remove the section about "Operator SDK", it will be supported in 3.11, but not 3.10.
2. Miss a section about "Remove etcd from Broker and move to using CRDs" and "Migration for openshift-ansible from etcd to crds", that is a big changes from OCP 3.9 to OCP 3.10.

Related Cards:
https://trello.com/c/K6zUdfLZ/668-21-remove-etcd-from-broker-and-move-to-using-crds
https://trello.com/c/KlFLeFI0/717-need-to-add-migration-path-for-openshift-ansible-from-etcd-to-crds

Related Doc:
https://github.com/openshift/ansible-service-broker/blob/master/docs/proposals/custom_resources_definitions.md

**Comment by chengzhang1016** [comment]
@ahardin-rh PR https://github.com/openshift/openshift-docs/pull/11122/files LGTM. Thanks.

**Comment by jianlinliu** [comment]
@ahardin-rh From the preview, I saw "System Container for installer and Kubelet" is marked as "Dropped", this is really a surprise to me. I did not find a correct to add my comment, so I add my question here.
For "System Container for installer", I did not heard this totally, and QE even have a card to indicate it will become fully supported (https://trello.com/c/1yeEnpHg/485-2-installer-as-a-system-container-fully-supported-oa-syscon-support), which one is true.
For "System Container for Kubelet", I did not heard this neither, but I saw related card (https://trello.com/c/hdUH6eXu/486-3-node-as-a-system-container-fully-supported-oa-syscon-support) is removed, my question if this is dropped, then how to set up cluster on Atomic Host, as far as I know, the env running on Atomic host, the node/kubelet is running as system container, do I miss something?

**Comment by jianlinliu** [comment]
For "Use of openshift_set_node_ip and openshift_ip Are No Longer Supported" part, refer to https://github.com/openshift/openshift-docs/pull/9702#issuecomment-393619514, should not we note customer about this issue after "Use of openshift_set_node_ip and openshift_ip Are No Longer Supported"?

**Pull request merged**